modules/react/layout/stories/examples/Grid/UIExample.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {Grid, GridProps, Box} from '@workday/canvas-kit-react/layout';
+import {type} from '@workday/canvas-kit-react/tokens';
+
+// temporary placeholder until type components are added to canvas-kit
+const H3 = props => (
```

Comment:
in other places, we just call this `StyledHeading` to indicate it's a styled element. I think you can accomplish most of this with as as well: `<Box as="h3"..../>` or create a styled `Box`